commands/reserves/reservableitem.js

```diff
@@ -0,0 +1,70 @@
+const { Op } = require("sequelize");
+
+// Add a new reservable item to the raid db
+exports.run = async function(client, message, args) {
+
+    if (!client.permission.manageReserves(message.member)) {
+		return message.channel.send('**Error:** You need permission to be able to add items to the reservable items list.');
+	}
+
+
+    if (!args[0]) {

```

Comment:
Instead of checking if the first arg is defined, can we check (args.length >= 3), to verify that all three arguments are set?